### PR TITLE
Add the CSV hash table in Hash layer and fix some bugs.

### PR DESCRIPTION
**Pull request by dengc367**
remove lambda function in Layer
add the csv HashTable lookup functionality in Hash Layer

### CODE REVIEW

**Comment by dengc367** [comment]
@shenweichen who can review this PR, I guess the hashtable functionality is usable for this library.

**Comment by shenweichen** [comment]
> @shenweichen who can review this PR, I guess the hashtable functionality is usable for this library.

can you provide an example  about using  **CSV hash table in Hash layer **  and add documentation in codes for this feature

**Comment by shenweichen** [comment]
> @shenweichen who can review this PR, I guess the hashtable functionality is usable for this library.

@dengc367  can you provide an example  about using  **CSV hash table in Hash layer **  and add documentation in codes for this feature.

File: deepctr/layers/utils.py [hunk 1]
```diff
@@ -30,9 +30,14 @@ class Hash(tf.keras.layers.Layer):
     if mask_zero = True,0 or 0.0 will be set to 0,other value will be set in range[1,num_buckets)
     """
 
-    def __init__(self, num_buckets, mask_zero=False, **kwargs):
+    def __init__(self, num_buckets, mask_zero=False, vocabulary_path=None, default_value=0, **kwargs):
```
Comment (shenweichen) [inline deepctr/layers/utils.py:33]: add some examples and docs about this feature

**Review by shenweichen — CHANGES_REQUESTED** [review]
please add some examples and docs about this feature.

**Comment by shenweichen** [comment]
please follow this contribution guide https://github.com/shenweichen/DeepCTR/blob/master/CONTRIBUTING.md

**Comment by dengc367** [comment]
OK. I will add some docs and examples shortly. @shenweichen

File: deepctr/layers/utils.py (hunk 1 above)
Comment (dengc367) [reply deepctr/layers/utils.py:61]: I add some docs and one example in the doc,  please review it.

**Comment by dengc367** [comment]
@shenweichen I add some docs, you can check and review it.

**Comment by dengc367** [comment]
> please follow this contribution guide https://github.com/shenweichen/DeepCTR/blob/master/CONTRIBUTING.md

OK. You can add me to the `Collaborator`. When I have spare time, I am grad to improve this project gradually.

**Comment by shenweichen** [comment]
> > please follow this contribution guide https://github.com/shenweichen/DeepCTR/blob/master/CONTRIBUTING.md
> 
> OK. You can add me to the `Collaborator`. When I have spare time, I am grad to improve this project gradually.

added

**Comment by shenweichen** [comment]
please follow our [contribution guide](https://github.com/shenweichen/DeepCTR/blob/master/CONTRIBUTING.md) so the continuous integration can run successfully

**Comment by dengc367** [comment]
OK. I have created another PR #385 , and so this PR need to be closed. Thanks.

**Pull request closed without merging**